openreview/conference/matching.py

```diff
@@ -61,6 +66,15 @@ def _build_entries(self, author_profiles, reviewer_profiles, paper_bid_jsons, sc
                     if bid_score != 0.0:
                         user_entry['scores']['bid'] = bid_score
 
+            reviewer_recommendations = [ t['tag'] for t in sorted(paper_recommendation_jsons, key=lambda x: x['cdate'])]
+            if reviewer_recommendations:
+                ## Set value between High(0.5) and Very High(1)
```

Comment:
this looks like it's calculating the score based on the default bid score mapping (the idea being that recommendations should be valued the same as "High" or "Very High" bids). This makes sense to me. But we might run into problems in the future if the user wants to change the bid score mapping.

I'm not sure if there's a good solution for this right now, so I'm not recommending any change. But we should keep an eye out for this in the future.

Reply:
I agree, both conversions functions should come from each conference matching configuration. We don't have that right now. I believe once we remove the metadata notes we can translate this the matching repo.